packages/edit-site/src/components/page-pages/style.scss

```diff
@@ -1,5 +1,28 @@
-.edit-site-page-pages__featured-image {
-	width: $grid-unit-50;
-	height: $grid-unit-50;
-	display: block;
+.dataviews-view-table {
```

Comment:
What do we need this for? I presume so these styles are only used in the table layout? Can we perhaps add the `.edit-site-page-pages_media-wrapper` class (or a different one) to the `span` in the page-pages/index.js component?

I think it's best if the consumers (Pages, Templates, etc.) don't use dataviews' classes, because they can change. In general, we avoid making classes "public API".

Reply:
You mean the specificity? Yes it's because the styles should only be applied in table layout. List layout already adds these styles on behalf of the consumer. 

> Can we perhaps add the .edit-site-page-pages_media-wrapper class (or a different one) to the span in the page-pages/index.js component only if the table layout is in use?

Let me see if I can do that.

Reply:
I can take a look at this and push to the branch tomorrow morning as well.